filesystem.go

```diff
@@ -152,11 +152,6 @@ func (fs *filesystem) createAllResources(pod Pod) error {
 }
 
 func (fs *filesystem) storeFile(path string, data interface{}) error {
-	_, err := os.Stat(path)
```

Comment:
@jodh-intel We need this function call to make sure we remove the existing file before we create it again. That's the way to override data stored in the filesystem.